yt_dlp/extractor/zdf.py

```diff
@@ -241,10 +242,23 @@ class ZDFIE(ZDFBaseIE):
             'title': 'Das Geld anderer Leute',
             'description': 'md5:cb6f660850dc5eb7d1ab776ea094959d',
             'duration': 2581.0,
-            'timestamp': 1654790700,
-            'upload_date': '20220609',
```

Comment:
Do you know why these changed? If not, make sure to check on the site that the new data is correct

Reply:
I think they re-aired the episode, changing the date. It also shows a different date on the website: [old](https://web.archive.org/web/20220610132401/https://www.zdf.de/serien/soko-stuttgart/das-geld-anderer-leute-100.html), [new](https://www.zdf.de/serien/soko-stuttgart/das-geld-anderer-leute-100.html)